parsers/pajtonparser/tadpole/pond.py

```diff
@@ -75,8 +76,12 @@ class PondOptions:
     distance: int
     number: bool
     collapse: bool
+    consecutive: bool
+
+    def __init__(self, distance: int, number=False, collapse=False, consecutive=False):
```

Comment:
Szczerze, to tych default argsów dla naszego use case'a nie trzeba, bo i tak argparser będzie defaultowo coś dawał, ale w perspektywie używania tego jako biblioteki jest good